lib/min_heap.rb

```diff
@@ -39,33 +44,63 @@ def to_s
     end
 
     output += @store.last.value + "]"
-      
+
     return output
   end
 
   # This method returns true if the heap is empty
-  # Time complexity: ?
-  # Space complexity: ?
+  # Time complexity: O(n)
+  # Space complexity: O(1)
   def empty?
-    raise NotImplementedError, "Method not implemented yet..."
+    if @store.empty?
+      return
+    end
```

Comment:
You're returning `nil` if `@store.empty?`  I would instead `return @store.empty?`